[2025-06-13T11:24:58.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-13T11:24:58.772+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exemplo_dag_simples.executar_funcao_python manual__2025-06-13T11:24:56.144536+00:00 [queued]>
[2025-06-13T11:24:58.782+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exemplo_dag_simples.executar_funcao_python manual__2025-06-13T11:24:56.144536+00:00 [queued]>
[2025-06-13T11:24:58.783+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-06-13T11:24:58.800+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): executar_funcao_python> on 2025-06-13 11:24:56.144536+00:00
[2025-06-13T11:24:58.807+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=267) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-13T11:24:58.807+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'exemplo_dag_simples', 'executar_funcao_python', 'manual__2025-06-13T11:24:56.144536+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/exemplo-dag.py', '--cfg-path', '/tmp/tmpvfvvx_v1']
[2025-06-13T11:24:58.809+0000] {standard_task_runner.py:72} INFO - Started process 268 to run task
[2025-06-13T11:24:58.810+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask executar_funcao_python
[2025-06-13T11:24:58.828+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-13T11:24:58.876+0000] {task_command.py:467} INFO - Running <TaskInstance: exemplo_dag_simples.executar_funcao_python manual__2025-06-13T11:24:56.144536+00:00 [running]> on host 3f818eaf31a0
[2025-06-13T11:24:58.976+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='exemplo_dag_simples' AIRFLOW_CTX_TASK_ID='executar_funcao_python' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T11:24:56.144536+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T11:24:56.144536+00:00'
[2025-06-13T11:24:58.977+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-13T11:24:58.992+0000] {logging_mixin.py:190} INFO - Executando DAG de exemplo no Airflow!
[2025-06-13T11:24:58.994+0000] {logging_mixin.py:190} INFO - Data e hora atual: 2025-06-13 11:24:58.993991
[2025-06-13T11:24:58.995+0000] {python.py:240} INFO - Done. Returned value was: Task executada com sucesso!
[2025-06-13T11:24:59.027+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-13T11:24:59.029+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=exemplo_dag_simples, task_id=executar_funcao_python, run_id=manual__2025-06-13T11:24:56.144536+00:00, execution_date=20250613T112456, start_date=20250613T112458, end_date=20250613T112459
[2025-06-13T11:24:59.066+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-13T11:24:59.102+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-13T11:24:59.107+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
